packages/teleport/src/Kubes/AddKube/AddKube.tsx

```diff
@@ -0,0 +1,235 @@
+              Step 2
+            </Text>
+            {
+              ' - Generate a script to automatically configure and install the teleport-agent'
+            }
+            <Validation>
+              {({ validator }) => (
+                <Flex alignItems="center" flexDirection="row">
+                  <form
+                    onSubmit={e => handleSubmit(e, validator)}
+                    style={{ width: '100%' }}
+                  >
+                    <FieldInput
+                      mb={2}
+                      rule={requiredField('Namespace is required')}
+                      label="Namespace"
+                      autoFocus
+                      value={namespace}
+                      placeholder="teleport"
+                      width="100%"
+                      mr="3"
+                      onChange={e => setNamespace(e.target.value)}
+                    />
+                    <FieldInput
+                      mb={2}
+                      rule={requiredField(
+                        'Kubernetes Cluster Name is required'
+                      )}
+                      label="Kubernetes Cluster Name"
+                      labelTip="Name shown to Teleport users connecting to the cluster."
+                      value={clusterName}
+                      placeholder="my-cluster"
+                      width="100%"
+                      mr="3"
+                      onChange={e => setClusterName(e.target.value)}
+                    />
+                    <ButtonPrimary
+                      block
+                      mt="2"
+                      disabled={attempt.status === 'processing'}
+                      type="submit"
+                    >
+                      {cmd ? 'Regenerate Script' : 'Generate Script'}
+                    </ButtonPrimary>
+                  </form>
+                </Flex>
+              )}
+            </Validation>
+          </Box>
+          {cmd && (
+            <Box mb={4}>
+              <Text bold as="span">
+                Step 3
+              </Text>
+              {' - Install the helm chart'}
+
+              <Box>
+                <Text mt="2" mb="1">
+                  The token will be valid for{' '}
+                  <Text bold as={'span'}>
+                    {token.expiryText}.
+                  </Text>
+                </Text>
+                <TextSelectCopy text={cmd} mb={2} />
+                <Text>
+                  <Text as="span" bold>
+                    Tip
+                  </Text>
+                  : Save the YAML file to apply updates later
+                </Text>
+              </Box>
+            </Box>
+          )}
+        </DialogContent>
+        <DialogFooter>
+          <ButtonSecondary onClick={onClose}>Close</ButtonSecondary>
+        </DialogFooter>
+      </Flex>
+    </Dialog>
+  );
+}
+const generateCmd = (
+  namespace: string,
+  clusterName: string,
+  proxyAddr: string,
+  token: string,
```

Comment:
As you're passing the `token.id` in...

```suggestion
  tokenId: string,
```